WordPress/Classes/ViewRelated/Stats/Traffic/StatsTrafficDatePickerView.swift

```diff
@@ -0,0 +1,91 @@
+import SwiftUI
+import DesignSystem
+
+struct StatsTrafficDatePickerView: View {
+    @ObservedObject var viewModel: StatsTrafficDatePickerViewModel
+
+    var body: some View {
+        HStack {
+            Menu {
+                ForEach([StatsPeriodUnit.day, .week, .month, .year], id: \.self) { period in
+                    Button(period.label, action: {
+                        viewModel.period = period
+                    })
+                }
+            } label: {
+                Text(viewModel.period.label)
+                    .style(TextStyle.bodySmall(.emphasized))
+                    .foregroundColor(Color.DS.Foreground.primary)
+                Image(systemName: "chevron.down")
+                    .font(.system(size: 8))
+                    .foregroundColor(Color.DS.Foreground.secondary)
+
+            }
+            .menuStyle(.borderlessButton)
+            .padding(.vertical, Length.Padding.single)
+            .padding(.horizontal, Length.Padding.double)
+            .background(Color.DS.Background.secondary)
+            .clipShape(RoundedRectangle(cornerRadius: Length.Radius.max))
+            .overlay(
+                RoundedRectangle(cornerRadius: Length.Radius.max)
+                    .strokeBorder(.clear, lineWidth: 0)
+            )
+
+            Spacer()
+
+            Text(viewModel.formattedCurrentPeriod())
+                .style(TextStyle.bodySmall(.emphasized))
+                .foregroundColor(Color.DS.Foreground.primary)
+                .lineLimit(1)
+
+            Spacer().frame(width: Length.Padding.single)
```

Comment:
Although designs have the spacing of `.single` (8) it visually has a larger spacing due to wider padding of buttons, both between text and arrow and between arrows themselves. So either we can increase the padding of arrow buttons or increase the spacing.

<img width="118" alt="image" src="https://github.com/wordpress-mobile/WordPress-iOS/assets/4062343/3ec978ab-2b7d-4ba4-a5a3-df8455081f92">

```suggestion
            Spacer().frame(width: Length.Padding.split)
```

Reply:
Good idea!